tools/c7n_mailer/c7n_mailer/email_delivery.py

```diff
@@ -18,12 +18,19 @@ def __init__(self, config, session, logger):
         self.config = config
         self.logger = logger
         self.session = session
-        self.aws_ses = session.client('ses', region_name=config.get('ses_region'))
+        self.provider = get_provider(self.config)
+        if self.provider == Providers.AWS:
+            self.aws_ses = session.client('ses', region_name=config.get('ses_region'))
```

Comment:
I added the following code here to utilise AWS SES to send mail. Please enlighten me with a better solution. Thanks.
```
        elif self.provider == Providers.GCP:
            self.aws_ses = boto3.client('ses', region_name=config.get('ses_region'))
```